src/ast/test/tests.cmake

```diff
@@ -0,0 +1,44 @@
+if(NOT EXISTS ${PARSER})
+  message(FATAL_ERROR " To run tests you must build the install target." ${PARSER})
+endif()
+
+find_package(Python3 COMPONENTS Interpreter)
+
+if (NOT Python3_FOUND)
+  message(WARNING " No python interpreter - Test Suite not compiled")
```

Comment:
nit: "no python 3" . Users might `python` and see that it's there, but it's python2.